src/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -35,7 +35,8 @@ internal class AcquireTokenCommonParameters
     {
         public ApiEvent.ApiIds ApiId { get; set; } = ApiEvent.ApiIds.None;
         public IEnumerable<string> Scopes { get; set; }
-        public Dictionary<string, string> ExtraQueryParameters { get; set; }
+        public IDictionary<string, string> ExtraQueryParameters { get; set; }
```

Comment:
Minor refactoring - we can accept an IDictionary for the extra query params instead of Dictionary.